infra-in-containers/resources/docker-compose.yml

```diff
@@ -0,0 +1,122 @@
+version: '2'
+services:
+  rainloop:
+    container_name: rainloop
+    image: hardware/rainloop
+    links:
+    - mail
+    volumes:
+    - rainloop_data:/rainloop/data
+  mail:
+    image: tvial/docker-mailserver:latest
+    hostname: ${HOSTNAME}
+    domainname: ${DOMAINNAME}
+    container_name: ${CONTAINER_NAME}
+    ports:
+    - "25:25"
+    - "143:143"
+    - "465:465"
+    - "587:587"
+    - "993:993"
+    - "4190:4190"
+    volumes:
+    - maildata:/var/mail
+    - mailstate:/var/mail-state
+    - ./config/:/tmp/docker-mailserver/
+    - ./ssl:/tmp/ssl:ro
+    environment:
```

Comment:
My personal take is that if there are this many env vars, put the config in a file instead.

Reply:
Done